src/envoy/tcp/mixer/filter.cc

```diff
@@ -167,6 +169,7 @@ void Filter::completeCheck(const CheckResponseInfo &info) {
 void Filter::onEvent(Network::ConnectionEvent event) {
   if (filter_callbacks_->upstreamHost()) {
     ENVOY_LOG(debug, "Called tcp filter onEvent: {} upstream {}",
+              filter_callbacks_->connection(),
```

Comment:
I think you need to add a third {} to the debug log

Reply:
Good catch. I think `ENVOY_CONN_LOG` should be used here.